core/templates/pages/exploration-editor-page/history-tab/services/version-tree.service.ts

```diff
@@ -144,7 +147,7 @@ export class VersionTreeService {
    *  - 'version_number': version number reverted to
    */
   getChangeList(version: number): ExplorationChange[] {
-    if (this._snapshots === null) {
+    if (this._snapshots === undefined) {
```

Comment:
Ditto, why is this needed?

Reply:
`_snapshots` is still uninitialized (undefined) at the top, so we need to add a check